bin/ChakraCore/ChakraCore.vcxproj

```diff
@@ -50,10 +50,10 @@
         advapi32.lib;
         ole32.lib;
         Rpcrt4.lib;
-	bcrypt.lib;
+        bcrypt.lib;
         $(ChakraCommonLinkDependencies)
       </AdditionalDependencies>
-      <AdditionalDependencies Condition="'$(UseICU)'=='true' AND '$(ChakraICU)'=='false'">
+      <AdditionalDependencies Condition="'$(ChakraICU)'!='false' AND '$(IcuLibraryDependencies)'!=''">
```

Comment:
When would ChakraICU be not false and IcuLibraryDependencies be empty? Seems like we should just fail the build if IcuLibraryDependencies are empty in this case

Reply:
IcuLibraryDependencies is empty when we are using static or shared, because `<ProjectReference>` takes care of linking to Chakra.ICU automagically. I am leaving this variable open for when we eventually need to allow bring-your-own-ICU.